src/Applications/LDAS_App/GEOSldas_LDAS.rc

```diff
@@ -114,6 +116,10 @@ FIRST_ENS_ID: 0
 #      Default is the entire simulation period (END_DATE minus BEG_DATE).
 #      Format:            yyyymmdd hhmmss
 #
+##  for adas-coupling exp ( COUPLING =1 )
+##  always use the fixed JOB_SGMT setting 6h ( as adas window) 
+##  JOB_SGMT: 00000000 060000
```

Comment:
Add check into setup routine that asserts that JOB_SGMT is 6 hours when COUPLING=1.
Exit setup with message that clarifies this.

Reply:
moved the text  to LADAS COUPLING EXP INFO  in ldas_setup. 
@weiyuan-jiang,  could you add the check/exit for this in ldas_setup ?

Reply:
It will reset the JOB_SGMT and issue warnings if JOB_SGMT is not 6h.  Change was pushed